ocd-2885.sql

```diff
@@ -0,0 +1,56 @@
+DROP SCHEMA IF EXISTS ff4j CASCADE;
+
+CREATE SCHEMA ff4j
```

Comment:
Why not just do the `\i dev/ff4j-schema.sql` file thing? This looks like that same code, or at least SUPER-CLOSE